### PR TITLE
Fix installing Cloud Pak for Data 4.0.2 services

### PR DESCRIPTION
**Pull request by jensmueller-com**
- Fix installing Cloud Pak for Data 4.0.2 services

### CODE REVIEW

File: docs/development_guide.md [hunk 1]
```diff
@@ -159,6 +159,14 @@ After having pushed the tag, the following actions are performed:
 - A Docker image containing the installed Python distribution is built and pushed to Quay.
 - A GitHub release draft is created, which must be manually published.
 
+## Adding support for a new IBM Cloud Pak for Data 4.0.x release
+
+To add support for a new IBM Cloud Pak for Data 4.0.x release, perform the following steps:
+
+- Update `CloudPakForDataManager._IBM_CLOUD_PAK_FOR_DATA_VERSION`
```
Comment (eirinikalogeiton) [inline docs/development_guide.md:166]: Please indicate the file that one can locate the `CloudPakForDataManager._IBM_CLOUD_PAK_FOR_DATA_VERSION`

Comment (eirinikalogeiton) [inline docs/development_guide.md:166]: Please indicate the file that one can locate the `CloudPakForDataManager._IBM_CLOUD_PAK_FOR_DATA_VERSION`

**Comment by eirinikalogeiton** [comment]
Hi @jensmueller-com . I propose to create a new configuration/constants file - if it is a json it is up to you - that have the parameters that could change for each service. 
I do not think the whole information of the `customresource` definition is needed in a new file, just the parameters that could change in each service. it should be like a global configuration file for services, so the developer does not need to iterate through 700 lines (as of now) of code. 

For instance, in case of datagate the `version` can change according to the release. You can also include there the storage option. So both of them are inputs variables to the  `"datagate": CustomResourceMetadata`. what do you think?

**Comment by jensmueller-com** [comment]
I am fine with a separate configuration file. However, I would prefer to have all information in one place as some of the other information might change as well:

```json
{
    "datagate": {
        "description": "Db2 Data Gate",
        "group": "datagate.cpd.ibm.com",
        "kind": "DatagateService",
        "licenses": [
            "Enterprise",
            "Standard"
        ],
        "name": "datagateservice-cr",
        "operator_name": "ibm-datagate-operator",
        "spec": {
            "datagate": "yes",
            "version": "2.0.2"
        },
        "status_key_name": "datagateStatus",
        "storage_option_required": true,
        "version": "v1"
    }
}
```

Would that be acceptable for you?

**Comment by eirinikalogeiton** [comment]
Hmm, what do you mean by a separate configuration file? Only for DG? Do you want to include the whole datagateservice CRD in a separate file? And if yes, then I do not see the point of this. If you do it for DG then you should do it for any service supported by the CLI, and I cannot find a benefit for that. 
As of now, the parameters that change in the DGservice is the version (in the specification) the version of the channel might change as well and the storage class (which is not provided in the script - I am guessing it comes from user input). So having a configuration file with options that may change makes sense to me. 
But if you do not want to do that, i.e. to have a separate configuration file with parameters that change, it is fine the way it is.

**Review by eirinikalogeiton — APPROVED** [review]

**Pull request merged**